models/models.articles.js

```diff
@@ -35,6 +35,26 @@ exports.selectArticleById = (id) => {
     });
 };
 
+exports.writeArticleById = (id, body) => {
+  const { inc_votes } = body;
+
+  if (typeof inc_votes === "number") {
+    return db
+      .query(
+        `UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *;`,
+        [inc_votes, id]
+      )
+      .then(({ rows }) => {
+        return rows;
+      });
+  } else {
```

Comment:
rather than checking that the votes are valid and rejecting all other queries, it would be better to check if the rows has come back empty and rejecting if it is